rainbows.rb

```diff
@@ -27,3 +28,13 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+rainbow = [{ red: {r: 255, g: 0, b: 0}}, { orange: {r: 255, g: 125, b: 0}},
+  { yellow: {r: 255, g: 255, b: 0}}, { green: {r: 0, g: 255, b: 0}}, { blue: {r: 0, g: 0, b: 255}},
+  { indigo: {r: 75, g: 0, b: 130}}, {violet: {r: 143, g: 0, b: 255}}]
```

Comment:
We went over the formatting notes I have about this in class. But summary: []'s on separate lines and each hash on a separate line.